vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -86,7 +86,7 @@ namespace
 namespace Microsoft::React
 {
 
-WinRTWebSocketResource::WinRTWebSocketResource(IMessageWebSocket&& socket, IDataWriter&& writer, Uri&& uri, vector<ChainValidationResult> &&certExeptions)
+WinRTWebSocketResource::WinRTWebSocketResource(IMessageWebSocket&& socket, IDataWriter&& writer, Uri&& uri, vector<ChainValidationResult> &&certExeptions) noexcept
   : m_uri{ std::move(uri) }
   , m_socket{ std::move(socket) }
   , m_writer{ std::move(writer) }
```

Comment:
(nit) usually, these follow same order as args for readability eg:
: m_socket{ std::move(socket) }
, m_writer{ std::move(writer) }
, m_uri{ std::move(uri) }

Reply:
Originally, I think I chose order of declaration in the header, but can also change this in a subsequent PR.